controllers/semeru_compiler.go

```diff
@@ -107,12 +129,73 @@ func (r *ReconcileWebSphereLiberty) reconcileSemeruDeployment(wlva *wlv1.WebSphe
 						},
 					},
 					Env: []corev1.EnvVar{
-						{Name: "OPENJ9_JAVA_OPTIONS", Value: "-XX:+JITServerLogConnections"},
+						{Name: "OPENJ9_JAVA_OPTIONS", Value: "-XX:+JITServerLogConnections -Xjit:vlog=/tmp/output.log"},
```

Comment:
@smcclem what is the default location of this log file (assuming it's output to a file)? Wondering if we should refer to the default location in probes instead of modifying the location.

Reply:
Okay. Yes, it would be a problem if the logging statement (checked in readiness probe) is removed from the log file due to rollover. We need to follow up on it. But we can proceed with merging this PR. Thanks.